packages/uploadthing/src/internal/handler.ts

```diff
@@ -224,10 +226,12 @@ export const buildRequestHandler = <
         input?: Json;
       };
 
-      await uploadable.resolver({
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+      const res = await uploadable.resolver({
         file: reqBody.file,
         metadata: reqBody.metadata,
       });
+      ee?.emit("callbackDone", res);
```

Comment:
Should this be prefixed to reduce chances of conflict with user code? 

Seems like you instantiate the EventEmitter though, so I am guessing this is not an issue? I haven't used EventEmitter before, so I am not 100% on the details of how it works.

Reply:
Me neither actually, but the eventlistener is just on this EventEmitter instance, so I'd assume it shouldnt conflict??

Reply:
Yeah that's what I was thinking.